### PR TITLE
feat: Add support for renaming tables in schema translator

### PR DESCRIPTION
**Pull request by vermas2012**
This is a bigtable-1.x merge for PR #3154
https://github.com/googleapis/java-bigtable-hbase/pull/3154

### CODE REVIEW

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/src/main/java/com/google/cloud/bigtable/hbase/tools/HBaseSchemaTranslator.java [hunk 1]
```diff
@@ -52,8 +57,7 @@
  * <p>Execute the following command to copy the schema from HBase to Cloud Bigtable:
  *
  * <pre>
- *   mvn exec:java \
- *  -Dexec.mainClass=com.google.cloud.bigtable.hbase.tools.HBaseSchemaTranslator \
+ * java -jar bigtable-hbase-1.x-tools-1.20.0-SNAPSHOT-jar-with-dependencies.jar com.google.cloud.bigtable.hbase.tools.HBaseSchemaTranslator \
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/src/main/java/com/google/cloud/bigtable/hbase/tools/HBaseSchemaTranslator.java:60]: The version number is old - I would replace this (and others below) with:
`java -jar bigtable-hbase-1.x-tools-<YOUR-VERSION-NUMBER>-jar-with-dependencies.jar ....`

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/pom.xml [hunk 2]
```diff
@@ -128,6 +133,14 @@
...
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>${compileSource.1.8}</source>
+          <target>${compileSource.1.8}</target>
+        </configuration>
+      </plugin>
```
Comment (igorbernstein2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/pom.xml:143]: this looks like a broken merge (its repeated above)

**Review by igorbernstein2 — CHANGES_REQUESTED** [review]

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/pom.xml (hunk 2 above)
Comment (vermas2012) [reply bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/pom.xml]: done.

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/src/main/java/com/google/cloud/bigtable/hbase/tools/HBaseSchemaTranslator.java [hunk 3]
```diff
@@ -69,8 +73,7 @@
  * <p>Run the tool from a host that can connect to HBase. Store HBase schema in a file:
  *
  * <pre>
- *   mvn exec:java \
- *  -Dexec.mainClass=com.google.cloud.bigtable.hbase.tools.HBaseSchemaTranslator \
+ * java -jar bigtable-hbase-1.x-tools-<your-version>-SNAPSHOT-jar-with-dependencies.jar com.google.cloud.bigtable.hbase.tools.HBaseSchemaTranslator \
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/src/main/java/com/google/cloud/bigtable/hbase/tools/HBaseSchemaTranslator.java:76]: please remove 'SNAPSHOT'

File: bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/src/main/java/com/google/cloud/bigtable/hbase/tools/HBaseSchemaTranslator.java [hunk 4]
```diff
@@ -81,8 +84,7 @@
  * Bigtable using the schema file:
  *
  * <pre>
- *   mvn exec:java \
- *  -Dexec.mainClass=com.google.cloud.bigtable.hbase.tools.HBaseSchemaTranslator \
+ * java -jar bigtable-hbase-1.x-tools-<your-version>-SNAPSHOT-jar-with-dependencies.jar com.google.cloud.bigtable.hbase.tools.HBaseSchemaTranslator \
```
Comment (kolea2) [inline bigtable-hbase-1.x-parent/bigtable-hbase-1.x-tools/src/main/java/com/google/cloud/bigtable/hbase/tools/HBaseSchemaTranslator.java:87]: please remove 'SNAPSHOT'

**Review by kolea2 — APPROVED** [review]
LGTM after small doc fixes

**Review by igorbernstein2 — APPROVED** [review]

**Pull request merged**